index.js

```diff
@@ -76,12 +81,12 @@ function handleCorsOptionsCallbackDelegator (optionsResolver, fastify, req, repl
       next(err)
     } else {
       const corsOptions = Object.assign({}, defaultOptions, options)
-      onRequest(fastify, corsOptions, req, reply, next)
+      handler(fastify, corsOptions, req, reply, next)
     }
   })
 }
 
-function onRequest (fastify, options, req, reply, next) {
+function handler (fastify, options, req, reply, next) {
```

Comment:
Dont name it handler. Name it addCorsHeaders or so.

Reply:
gonna call it `addCorsHeadersHandler` since there's a function already named `addCorsHeaders`, and the word `handler` is good to have since it's mainly called from a hook handler (like `onRequest`)